src/thrift/parquet.thrift

```diff
@@ -191,6 +191,30 @@ enum FieldRepetitionType {
   REPEATED = 2;
 }
 
+enum BloomFilterStrategy{
+  /**
+   * See "Less Hashing, Same Performance: Building a Better Bloom Filter" by Adam Kirsch and
```

Comment:
Rather than referring to a paper, this should explain what MURMUR128_32 means. We can provide more detail in the Statistics.md page, but I don't want the spec to depend on reading this paper to know how to build or use the bloom filter.